docs/SetupDatabaseForAzure.md

```diff
@@ -0,0 +1,56 @@
+# Set up a database for Scalar DB/Scalar DL deployment in Azure
+
+This guide explains how to set up a database for Scalar DB/Scalar DL deployment in Azure.
+
+## Cosmos DB
+
+In this section, you will create a Cosmos DB account.
+
+### Requirements
+
+* You must create a Cosmos DB account with the `Core(SQL)` API.
+* You must change Cosmos DB `Default consistency` to `Strong`.
+* You must use Capacity mode as `Provisioned throughput`.
+
+### Recommendations
+
+* You should configure the backup policy as `Continuous` for Point-in-time-restore (PITR).
+* You should configure the service endpoint based on the [Azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/how-to-configure-vnet-service-endpoint) to restrict access to Cosmos DB for production.
+
+### Steps
+
+* Create an Azure Cosmos DB account based on the [Azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/create-cosmosdb-resources-portal#create-an-azure-cosmos-db-account) with the above requirements.
+
+Optional steps:-
```

Comment:
I think we can specify `(optional at the beginning of steps)`.
example:-
* (Optional) Configure advanced monitoring services with [Monitor Azure Cosmos DB](https://docs.microsoft.com/en-us/azure/cosmos-db/monitor-cosmos-db) guide, by default monitoring (metrics) will be enabled in Cosmos DB.

Ref: https://kubernetes.io/docs/reference/access-authn-authz/authentication/
https://docs.aws.amazon.com/step-functions/latest/dg/sample-eks-cluster.html

I don't think it will be redundant.

Reply:
Thank you.
I will update the document based on your suggestion.